plugins/database/mongodb/util.go

```diff
@@ -37,3 +51,21 @@ func (roles mongodbRoles) toStandardRolesArray() []interface{} {
 	}
 	return standardRolesArray
 }
+
+// mgo.Role is a named string type
+func (roles mongodbRoles) toStandardRolesStringArray() []mgo.Role {
```

Comment:
This func doesn't seem to be used anywhere.

Reply:
Ah great catch! It was early on, but I was able to refactor and no longer used it, and just forgot. Thanks! I've removed it